Dockerfile-caveman-slim.template

```diff
@@ -13,6 +13,7 @@ ENV PYTHONIOENCODING UTF-8
 RUN apt-get update && apt-get install -y --no-install-recommends \
 		ca-certificates \
 		netbase \
+		libbluetooth-dev \
```

Comment:
This should be down in the temporary list (not this persistent list), like in `Dockerfile-slim.template` -- also, is it confirmed that Python 2 does support this as well or is it Python 3 only?

Reply:
Doh, just saw your note ("This change does not apply to 2.x, but since 2.x is deprecated in 9 days anyway") -- so this should be removed from all `-caveman` templates which should remove it from all 2.x versions. :+1: